app/views/layouts/countdown.html.erb

```diff
@@ -1,56 +0,0 @@
-<!DOCTYPE html>
-<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
```

Comment:
Couldn't find anywhere that referenced this view - seems to be old and no longer used.